lib/common.sh

```diff
@@ -308,7 +308,13 @@ fi
 export KIND_NODE_IMAGE=${KIND_NODE_IMAGE:-"${DOCKER_HUB_PROXY}/kindest/node:${KIND_NODE_IMAGE_VERSION}"}
 
 # Ansible version
-export ANSIBLE_VERSION=${ANSIBLE_VERSION:-"7.0.0"}
+# Older ubuntu version do no support 7.0.0 because of older python versions
+# Ansible 7.0.0 requires python 3.10+
+if [[ ${DISTRO} == "ubuntu22" ]]; then
+    export ANSIBLE_VERSION=${ANSIBLE_VERSION:-"7.0.0"}
+else 
+    export ANSIBLE_VERSION=${ANSIBLE_VERSION:-"6.5.0"}
```

Comment:
6.6.0 seems to be latest 6.x, could we use that?

Reply:
previous one we had 6.5.0, and we updated to 7.0.0.  We normally test each version of ansible, so sticking to  6.5.0 version is preferred, since 6.6.0 is not tested.

Reply:
Agree with @Sunnatillo